tgui/packages/tgui/interfaces/Bombsim/index.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Romayne (https://github.com/MeggalBozale)
+ * @license ISC
+ */
+
+import { useBackend, useLocalState } from '../../backend';
+import { Button, LabeledList, Section, Stack } from '../../components';
+import { Window } from '../../layouts';
+import { formatPressure, truncate } from '../../format';
+
+const TankDisplay = (props, context) => {
```

Comment:
Why not just use the tank display that TTVs/canisters use, for parity

Reply:
ill attempt that and post how it looks :+1: